pam/integration-tests/gdm_test.go

```diff
@@ -443,10 +443,6 @@ func TestGdmModule(t *testing.T) {
 					gdm_test.IsAuthenticatedEvent(&authd.IARequest_AuthenticationData_Secret{
 						Secret: "noble2404",
 					}),
-					// Use previous one
-					gdm_test.IsAuthenticatedEvent(&authd.IARequest_AuthenticationData_Secret{
-						Secret: "goodpass",
-					}),
```

Comment:
Can you add a passwd test here? It's technically not used (yet), but we may end up using it once when we'll properly implement the new password by going through PAM.

Reply:
I already looked into that but AFAICT the `gdmTestModuleHandler` which is used here does not support passwd mode.

Reply:
Yeah, right... Since it's not something we actually care in gdm, since it cannot go in passwd mode anyways.